lib/vm/src/instance/mod.rs

```diff
@@ -625,7 +625,7 @@ impl Instance {
         // https://webassembly.github.io/reference-types/core/exec/instructions.html#exec-elem-drop
 
         let mut passive_elements = self.passive_elements.borrow_mut();
-        passive_elements.remove(&elem_index);
+        passive_elements[elem_index] = Box::new([]);
```

Comment:
Yeah I was thinking about that. I think perhaps a better structure is `MaybeUninit` (instead of Box), as it will probably require no changes on the access from the generated compiled code from the compilers.

Reply:
`Option<Box<T>>` is FFI safe! It's in fact 100% identical to `Box<T>`! We use it in the C API all the time.

That said, I actually don't know if `Box<[T]>` in general has a defined layout but 🤷

Reply:
(The reason is that `Box<T>` can never be null. `Option<Box<T>>` is just how you say "nullable (heap) pointer" in Rust. Rust optimizes that and `None` is equal to nullptr and Some(ptr) is normal `Box<T>` internally)

Reply:
Awesome!